iOS/IntercomWrapper.m

```diff
@@ -156,5 +156,12 @@ @implementation IntercomWrapper
     callback(@[[NSNull null]]);
 };
 
+// Available as NativeModules.IntercomWrapper.setHMAC
+RCT_EXPORT_METHOD(setHMAC:(NSString*)hmac data:(NSString*)data callback:(RCTResponseSenderBlock)callback) {
+    NSLog(@"setHMAC with %@", hmac);
```

Comment:
Certainly! I didn't like them but wanted to be consistent with the existing methods.